libraries/AP_HAL_ChibiOS/QSPIDevice.cpp

```diff
@@ -45,16 +45,22 @@ static const struct QSPIDriverInfo {
 QSPIDesc QSPIDeviceManager::device_table[] = { HAL_QSPI_DEVICE_LIST };
 
 // Check clock sanity during runtime
-#define XSTR(x) STR(x)
-#define STR(x) #x
+#define QUOTE(str) #str
```

Comment:
This doesn't seem to be getting used here, best to remove.

Reply:
yes

Reply:
removed